test/runtests.jl

```diff
@@ -402,18 +405,8 @@ end
 
     if is_local
         flavours = ["os", "ptp", "ata"]
```

Comment:
```suggestion
```
I guess this line could be deleted, as we are not testing the `flavours` here any more.